includes/Admin/Settings_Screens/Messenger.php

```diff
@@ -156,6 +157,34 @@ public function get_settings() {
 	}
 
 
+	/**
+	 * Gets the upcoming deprecation warning notice.
+	 *
+	 * @since x.x.x
+	 *
+	 * @return string
+	 */
+	public function get_deprecation_warning() {
+		return '<div class="notice notice-warning"><p>' . self::get_deprecation_message() . '</p></div>';
+	}
+
+	/**
+	 * Gets the upcoming deprecation warning message with documentation link.
+	 *
+	 * @since x.x.x
+	 *
+	 * @return string
+	 */
+	public static function get_deprecation_message() {
+		return sprintf(
+			/* translators: Placeholders: %1$s - <a> tag, %2$s - </a> tag */
+			__( '️<strong>Facebook for WooCommerce</strong>: Since %1$sMeta plans to discontinue the Chat Plugin%2$s, the Messenger feature will be removed by May, 2024.', 'facebook-for-woocommerce' ),
+			'<a href="https://href.li/?https://www.facebook.com/business/help/1524587524402327" target="_blank">',
+			'</a>'
+		);
```

Comment:
I noticed the URL you shared and the href.li redirect and also didn't show the notice. So I suggest updating this to the about page which seem to show the notice consistently.

```suggestion
			/* translators: Placeholders: %1$s - <a> tag, %2$s - </a> tag */
			__( '️<strong>Facebook for WooCommerce</strong>: Since %1$sMeta plans to discontinue the Chat Plugin%2$s, the Messenger feature will be removed by May, 2024.', 'facebook-for-woocommerce' ),
			'<a href="https://www.facebook.com/business/help/1661027437357021" target="_blank">',
			'</a>'
		);
```

Reply:
Yeah, that's probably a better source of truth

Reply:
Updated https://github.com/woocommerce/facebook-for-woocommerce/pull/2708/commits/7f35cdefb121d1e0d33371ce2c0a7e94980405a6